Omikron/Factfinder/Helper/Data.php

```diff
@@ -130,12 +142,12 @@ public function getShowAddToCartButton()
 
     /**
      * Returns the current FACT-Finder version
-     * @param null|string $storeCode
+     * @param null|int|string $scopeCode
```

Comment:
I switched to naming it scopeCode because scopeConfig->getValue(...) uses it as scopeCode too and i tried to align the naming to that function.
scopeConfig->getValue(...) gets the value from the database table "core_config_data".  This table uses the storeId for its field "scope_id". 

public function getValue($path, $scopeType = ScopeConfigInterface::SCOPE_TYPE_DEFAULT, $scopeCode = null);

On the other hand storeCode or storeId (or even scopeId) might be better understandable...I decided in the end for scopeCode, but there is nothing wrong with the other names.